crates/router/src/configs/settings.rs

```diff
@@ -393,6 +396,24 @@ pub struct NetworkTransactionIdSupportedConnectors {
     pub connector_list: HashSet<enums::Connector>,
 }
 
+#[derive(Debug, Deserialize, Clone, Default)]
+pub struct NetworkTokenizationSupportedCardNetworks {
+    #[serde(deserialize_with = "deserialize_hashset")]
+    pub card_networks: HashSet<enums::CardNetwork>,
+}
+
+#[derive(Debug, Deserialize, Clone)]
+pub struct NetworkTokenizationService {
```

Comment:
Add comments, explaining about the fields

Reply:
this is already explained in env_specific.toml